packagemanager/pypi.go

```diff
@@ -137,10 +175,114 @@ func (pip *pipPackageManager) ParseCommand(args []string) (*ParsedCommand, error
 	}, nil
 }
 
-// pipParsePackageInfo parses a pip install package specification, separating the package name,
+type uvCommandParser struct {
+	config PypiPackageManagerConfig
+}
+
+func NewUVCommandParser(config PypiPackageManagerConfig) pypiCommandParser {
+	return &uvCommandParser{
+		config: config,
+	}
+}
+
+func (u *uvCommandParser) ParseCommand(args []string) (*ParsedCommand, error) {
+	// Remove 'uv' if it's the first argument
+	if len(args) > 0 && args[0] == "uv" {
+		args = args[1:]
+	}
+
+	command := Command{Exe: u.config.CommandName, Args: args}
+	if len(args) < 1 {
+		return &ParsedCommand{Command: command}, nil
+	}
+
+	// Handle uv sync command (installs from uv.lock)
+	if args[0] == "sync" {
+		return &ParsedCommand{
+			Command:           command,
+			InstallTargets:    nil,
+			IsManifestInstall: true,
+			ManifestFiles:     []string{"uv.lock"},
+		}, nil
+	}
+
+	// Find the install command position
+	var installCmdIndex = -1
+	for idx, arg := range args {
+		if slices.Contains(u.config.InstallCommands, arg) {
+			installCmdIndex = idx
+			break
+		}
+	}
+
+	if installCmdIndex == -1 {
+		// No install command found, return as-is
+		return &ParsedCommand{Command: command}, nil
+	}
+
+	// Extract arguments after the install command
+	installArgs := args[installCmdIndex+1:]
+
+	// Set up flag parsing
+	fs := pflag.NewFlagSet("uv", pflag.ContinueOnError)
```

Comment:
Nice. Like the approach of using `pflag` for arg parsing. But `fs` seems like a misleading variable name. Initially I thought `FileSystem`